app/src/main/java/at/irfc/app/ui/map/MapScreen.kt

```diff
@@ -1,30 +1,76 @@
 package at.irfc.app.ui.map
 
-import androidx.compose.foundation.background
+import androidx.compose.foundation.Image
+import androidx.compose.foundation.gestures.detectTransformGestures
 import androidx.compose.foundation.layout.*
-import androidx.compose.material3.MaterialTheme
-import androidx.compose.material3.Text
-import androidx.compose.runtime.Composable
-import androidx.compose.ui.Alignment
+import androidx.compose.runtime.*
 import androidx.compose.ui.Modifier
-import androidx.compose.ui.res.stringResource
-import androidx.compose.ui.unit.dp
+import androidx.compose.ui.draw.clipToBounds
+import androidx.compose.ui.geometry.Offset
+import androidx.compose.ui.graphics.graphicsLayer
+import androidx.compose.ui.input.pointer.pointerInput
+import androidx.compose.ui.layout.ContentScale
+import androidx.compose.ui.layout.onSizeChanged
+import androidx.compose.ui.res.painterResource
+import androidx.compose.ui.unit.IntSize
 import at.irfc.app.R
 import com.ramcosta.composedestinations.annotation.Destination
 
+const val MIN_SCALE = 1f
+const val MAX_SCALE = 3f
+const val ASPECT_RATIO = 4 / 3f
```

Comment:
Make them private please, otherwise they pollute the global namespace. Or even better make them parameters of the ZoomableImage function, that way it would be reusable.

Reply:
implemented as parameter, done